xml/release-notes.xml

```diff
@@ -648,6 +648,70 @@ ServerArguments=-nolisten tcp -dpi <replaceable>DPI_SETTING</replaceable>
    </para>
   </sect2>
 
+  <sect2 xml:id="sec.desktop.qt-hidpi">
+   <!-- boo#1091103 -->
+   <title>Using Automatic Scaling in Qt Applications in Setups Which Mix High-DPI/Regular-DPI Monitors</title>
+   <para>
+    Qt supports automatic per-monitor scaling on X. It uses the DPI value of
+    the virtual X screen to calculate the font size for the primary monitor.
+    By default, this value is 96 DPI. It uses the relative DPI of the primary
+    monitor to derive font DPI for all other monitors.
+   </para>
+   <para>
+    Two widely used desktops will override this behavior of Qt, therefore this
+    note does not apply to them:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      GNOME will set DPI according to each monitor's reported scaling.
```

Comment:
This is wrong. GNOME sets Xft.dpi to 96 * the manually configured scale (100%/200%/300%)